src/term_like.rs

```diff
@@ -76,4 +81,44 @@ impl TermLike for Term {
     fn flush(&self) -> io::Result<()> {
         self.flush()
     }
+
+    fn supports_ansi_codes(&self) -> bool {
+        static SUPPORTS_ANSI: OnceCell<bool> = OnceCell::new();
```

Comment:
I noticed the use of static here is incorrect, and I don't see an easy way around that.